docs/CONTRIBUTING.md

```diff
@@ -168,7 +178,8 @@ setup properly.
 
 While not strictly required, we use [asdf-vm](https://asdf-vm.com) to manage
 language versions for Firezone. You'll need to install the language runtimes
-according to the versions laid out in the [.tool-versions](../.tool-versions) file.
+according to the versions laid out in the [.tool-versions](../.tool-versions)
+file.
```

Comment:
(Had to split this up because Github said I can't make suggestions on deleted lines 🤷‍♀️ )
```suggestion
using the [.tool-versions](../.tool-versions)
file.
```

Reply:
```suggestion
defined in the [.tool-versions](../.tool-versions) file.
```

Maybe this is a little more clear?

Reply:
👍